packages/components/src/Tabs.tsx

```diff
@@ -6,18 +6,24 @@ export interface TabsProps {
   activeIndex?: number;
   children: Array<React.ReactElement<TabProps>>;
   TabComponent?: any;
+  TabsNavComponent?: any;
 }
 
 export interface TabsState {
   activeIndex: number;
 }
 
-const StyledUl = styled.ul`
+const StyledNav = styled.nav`
+  border-bottom: 1px solid #d8d8d8;
+`;
+
+const TabContainer = styled.ul`
+  display: flex;
   list-style: none;
+  margin: 0 auto;
+  max-width: 1200px;
```

Comment:
Should we have a `max-width` set at the component level?

I'm thinking we rely on `width: 100%` and have the parent container specify the width. That should make it more reusable.